### PR TITLE
dolt-commit: add page

### PR DESCRIPTION
**Pull request by bl-ue**
For #5471

### CODE REVIEW

File: pages/common/dolt-commit.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`dolt commit --message "{{commit_message}}"`
+
+- Stage all unstaged changes to tables before committing:
+
+`dolt commit --all`
+
+- Use the specified commit date (defaults to current date and time):
+
+`dolt commit --date "{{date}}"`
```
Comment (marchersimon) [inline pages/common/dolt-commit.md:20]: Could you maybe give an example, so it is clear in which format the date has to be specified?

Comment (bl-ue) [reply pages/common/dolt-commit.md:20]: It supports ISO 8601 dates. How about this? `2021-01-31T00:00:00`

Comment (marchersimon) [reply pages/common/dolt-commit.md:20]: Sure, looks good.

**Review by marchersimon — APPROVED** [review]

**Review by yutyo — APPROVED** [review]
LGTM.

**Comment by bl-ue** [comment]
@yutyo, remember, when you merge a PR, check if the PR links to an issue (like #5471, as this one does), and check the checkbox(es) that the PR fixes.

**Pull request merged**